yarn.lock

```diff
@@ -15475,6 +15475,11 @@ is-absolute-url@^3.0.0:
   resolved "https://registry.yarnpkg.com/is-absolute-url/-/is-absolute-url-3.0.1.tgz#e315cbdcbbc3d6789532d591954ac78a0e5049f6"
   integrity sha512-c2QjUwuMxLsld90sj3xYzpFYWJtuxkIn1f5ua9RTEYJt/vV2IsM+Py00/6qjV7qExgifUvt7qfyBGBBKm+2iBg==
 
+is-absolute-url@^3.0.2:
```

Comment:
This dependency seems to not be great at its job: https://github.com/sindresorhus/is-absolute-url/blob/master/index.js

```
/^[a-z][a-z\d+\-.]*:/.test('http://attacker.com')
true
/^[a-z][a-z\d+\-.]*:/.test('Http://attacker.com')
false
```